assets/collapsible-content.css

```diff
@@ -0,0 +1,117 @@
+.collapsible-section-layout {
```

Comment:
We should review the file naming conventions in a separate PR. I would expect something like `section-collapsible-content.css` for consistency.